mage_ai/services/spark/config.py

```diff
@@ -4,4 +4,4 @@
 
 @dataclass
 class SparkConfig(BaseConfig):
-    repo_path: str = None
+    spark_config: dict = None
```

Comment:
why don't we put the config fields in this dataclass?
for example
```
@dataclass
class SparkConfig(BaseConfig):
    app_name: str = None
    spark_master: str = None
    spark_home: str = None
    executor_env: Dict = field(default_factory=dict)
    spark_jars: List = field(default_factory=list)
    others: Dict = field(default_factory=dict)
```

Reply:
Good idea: made the suggested change.